api/tests/routes/backoffice_v3/pro_support_test.py

```diff
@@ -0,0 +1,195 @@
+
+        self._assert_error_400(
+            authenticated_client, venue, venue, expected_alert="Les lieux source et destination doivent être différents"
+        )
+
+    @clean_database
+    def test_move_siret_not_matching_venue(self, authenticated_client):
+        # given
+        offerer = offerers_factories.OffererFactory(siren="123456789")
+        venue1 = offerers_factories.VenueFactory(managingOfferer=offerer, siret="12345678900001")
+        venue2 = offerers_factories.VirtualVenueFactory(managingOfferer=offerer, siret=None)
+
+        self._assert_error_400(
+            authenticated_client,
+            venue1,
+            venue2,
+            siret="12345678900002",
+            expected_alert=f"Le SIRET 12345678900002 ne correspond pas au lieu {venue1.id}",
+        )
+
+    @clean_database
+    def test_move_siret_virtual_venue(self, authenticated_client):
+        offerer = offerers_factories.OffererFactory(siren="123456789")
+        venue1 = offerers_factories.VenueFactory(managingOfferer=offerer, siret="12345678900001")
+        venue2 = offerers_factories.VirtualVenueFactory(managingOfferer=offerer, siret=None)
+
+        self._assert_error_400(
+            authenticated_client,
+            venue1,
+            venue2,
+            expected_alert=f"Le lieu cible {venue2.name} (ID {venue2.id}) est un lieu virtuel",
+        )
+
+    @clean_database
+    def test_move_siret_different_offerer(self, authenticated_client):
+        venue1 = offerers_factories.VenueFactory(siret="12345678900001")
+        venue2 = offerers_factories.VenueFactory(siret=None, comment="No SIRET")
+
+        self._assert_error_400(
+            authenticated_client,
+            venue1,
+            venue2,
+            expected_alert=f"Source {venue1.id} and target {venue2.id} do not have the same offerer",
+        )
+
+    @clean_database
+    def test_move_siret_target_with_siret(self, authenticated_client):
+        offerer = offerers_factories.OffererFactory(siren="123456789")
+        venue1 = offerers_factories.VenueFactory(managingOfferer=offerer, siret="12345678900001")
+        venue2 = offerers_factories.VenueFactory(managingOfferer=offerer, siret="12345678900003")
+
+        self._assert_error_400(
+            authenticated_client,
+            venue1,
+            venue2,
+            expected_alert=f"Target venue {venue2.id} already has a siret: {venue2.siret}",
+        )
+
+    @clean_database
+    def test_move_siret_high_revenue(self, authenticated_client):
+        offerer = offerers_factories.OffererFactory(siren="123456789")
+        venue1 = offerers_factories.VenueFactory(managingOfferer=offerer, siret="12345678900001")
+        venue2 = offerers_factories.VenueFactory(managingOfferer=offerer, siret=None, comment="No SIRET")
+        bookings_factories.ReimbursedBookingFactory.create_batch(
+            34, venue=venue2, quantity=1, stock__price=300, stock__offer__venue=venue2
+        )
```

Comment:
Sur ma machine, ce test prend 4 secondes. Je l'ai réduit à environ 1,5s en créant une seule réservation d'un montant très élevé, réservé par un riche bénéficiaire : 

```suggestion
        rich_beneficiary = users_factories.BeneficiaryGrant18Factory(deposit__amount=100_000)
        bookings_factories.ReimbursedBookingFactory(
            user=rich_beneficiary,
            stock__price=10200,
            stock__offer__venue=venue2,
        )
```

Idem dans `test_move_siret_ok()`.

(Autre détail : l'argument `venue` donné à `BookingFactory` est ignoré. La factory le déduit automatiquement de `stock.offer.venue`, cf. [code de la factory](https://github.com/pass-culture/pass-culture-main/blob/39f5d066a25f38d1ec9446e7d73584cbe9b4def4/api/src/pcapi/core/bookings/factories.py#L72).)

Reply:
OK, je ne pensais pas que le modèle permettait d'attribuer un crédit si élevé